Documentation/admin/install-multi.md

```diff
@@ -0,0 +1,205 @@
+
+## Install the registry
+
+Install the multi-cluster registry by downloading and applying the following manifests:
+* [cluster-crd.yaml][cluster-crd]
+* [cluster-view-role-binding.yaml][cluster-binding]
+* [cluster-view-role.yaml][cluster-role]
+* [directory-sync-account.yaml][directory-account]
+* [directory-sync-deployment.yaml][directory-deployment]
+* [directory-sync-role-binding.yaml][directory-binding]
+
+
+Once downloaded, use `kubectl apply` to install each in turn:
+
+```
+kubectl apply --record -f <url to manifest>
+```
+
+## Set up the directory cluster
+
+Select one of your clusters to be the "directory cluster", which is the cluster that is the source of truth for your policies. Other clusters will be configured to connect to this one. Check that kubectl is set up correctly and can talk to the cluster:
+
+```
+export KUBECONFIG=/path/to/kubeconfig
+kubectl get nodes
+```
+
+Next, read back the list of Clusters. There shouldn’t be any yet:
+
+```
+$ kubectl get clusters
+No resources found.
+```
+
+Create a new Cluster object that represents this cluster. Make sure this matches the name you installed with:
+
+```
+kubectl --namespace=tectonic-system get configmap/tectonic-config -o jsonpath="{..data.clusterName}"
+west-coast-production
+```
+
+Give the cluster a set of labels. These labels will be used in Cluster Policies:
+
+```
+apiVersion: multicluster.coreos.com/v1
+kind: Cluster
+metadata:
+  name: west-coast-production
+  labels:
+    stage: production
+    cloud: aws
+    region: us-west-1
+    role: replica
+  annotations:
+    multicluster.coreos.com/console-url: https://west-coast-production.west.example.com
+    multicluster.coreos.com/directory: true
+spec:
+  KubernetesAPIEndpoints:
+    ServerEndpoints:
+      - ServerAddress: https://west-coast-production-api.west.example.com:443
+```
+
+Last, save the file and submit it to the cluster:
+
+```
+kubectl create -f clusters/west-coast-production.yaml
+cluster “west-coast-production” created
+```
+
+Before setting up a replica cluster, register it in the directory. Change the labels and endpoints as necessary. Be sure that  the `multicluster.coreos.com/directory` annotation is only included on the directory cluster. Submit it to the cluster:
+
+```
+kubectl create -f clusters/east-coast-production.yaml
+cluster “east-coast-production” created
+```
+
+List the clusters:
+
+```
+kubectl get clusters
+NAME                    AGE
+west-coast-production   1m
+east-coast-production   10s
+```
+
+The directory cluster is all set up!
+
+## Generate credentials
+
+Next, create credentials for replica clusters to read the directory objects. To do this, use a service account created on the directory cluster, then export it for use on the replica cluster.
+
+Create a Service Account for the replica cluster. By convention, give the Service Account the same name as the replica cluster.
+
+```
+kubectl create serviceaccount -n tectonic-multi-cluster east-coast-production
```

Comment:
s/create/apply/g

Reply:
changing the command should change the return as well, right? will they all now follow the format
`
$ kubectl apply -f docs/user-guide/nginx/nginx-deployment.yaml
deployment "my-nginx" configured
`

where "created" changes to "configured" for each return?